src/DataMapper/OrderDataMapper.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusFacebookPlugin\DataMapper;
+
+use FacebookAds\Object\ServerSide\Content;
+use Setono\SyliusFacebookPlugin\Formatter\MoneyFormatterInterface;
+use Setono\SyliusFacebookPlugin\ServerSide\ServerSideEventInterface;
+use Sylius\Component\Core\Model\OrderInterface as CoreOrderInterface;
+use Sylius\Component\Core\Model\OrderItemInterface as CoreOrderItemInterface;
+use Sylius\Component\Order\Model\OrderInterface;
+use Sylius\Component\Order\Model\OrderItemInterface;
+use Webmozart\Assert\Assert;
+
+/** not final */ class OrderDataMapper implements DataMapperInterface
```

Comment:
What do you think about making these final and then in the future if anybody requests it, we can make them non final?

Reply:
Data mappers designed to be overriden.
They have protected methods to get additional emails/phones for example